MODULE.bazel

```diff
@@ -2,6 +2,15 @@ bazel_dep(name = "aspect_bazel_lib", version = "2.15.3")
 bazel_dep(name = "libarchive", version = "3.8.1")
 bazel_dep(name = "hermetic_cc_toolchain", version = "4.0.0")
 
+# Apply patches to libarchive for Windows fixes
+single_version_override(
+    module_name = "libarchive",
+    patches = [
+        "//patches:archive_read_support_format_mtree.patch",
```

Comment:
please update the filename to make it easier to find the upstream PR on libarchive, and/or link to that PR in a TODO to remove it after that's merged